main.js

```diff
@@ -27,23 +30,105 @@ class ModuleInstance extends InstanceBase {
 		this.config = config
 	}
 
+	async sendCam(self, str) {
+		const { default: got } = await import('got');
+		if (str) {
+
+			const url = `https://${self.config.httpHost}:${self.config.httpPort}${str}`
```

Comment:
this is super generic, why call the function postCam and then use only a generic url. Suggest to write only one function with a generic url and also a generic name and call that from more specialized functions (dry principle)